etc/tiny-dfr.conf

```diff
@@ -1,15 +1,99 @@
 [ui]
-primary_layer = "function"
-secondary_layer = "special"
+primary_layer = "specialextended"
+secondary_layer = "function"
 font = "sans-serif"
-icon_theme = "tiny-dfr-icons"
-
-[apps]
-app_icon_theme = "tiny-dfr-icons"
-app1_icon = "app1"
-app2_icon = "app2"
-app3_icon = "app3"
-app4_icon = "app4"
+media_icon_theme = "tiny-dfr-icons"
+app_icon_theme = "breeze"
```

Comment:
App_icon_theme in the repo should be hicolor imo, it’s a theme available in all distros

Reply:
done

Reply:
tested the code. working